spec/spec_helper.rb

```diff
@@ -102,3 +114,15 @@ def configure_rails_for_test
     puts SpecLogger.read
   end
 end
+
+RSpec.shared_context 'stubbed_central_config' do
+  before(:all) do
+    WebMock.stub_request(
+        :get, %r{^http://localhost:8200/config/v1/agents/?$}
+    ).to_return(body: '{}')
+  end
+
+  after(:all) do
+    WebMock.reset!
```

Comment:
After this PR we can use the regular `webmock/rspec` require and not have to do this.